modules/simple_forms_api/app/models/simple_forms_api/vba_21_0966.rb

```diff
@@ -67,7 +67,12 @@ def submission_date_config
     def track_user_identity(confirmation_number)
       identity = data['preparer_identification']
       StatsD.increment("#{STATS_KEY}.#{identity}")
+      benefit_types = data['benefit_selection'].map do |benefit_type, is_selected|
+        benefit_type if is_selected
+      end.compact.join(', ')
+
       Rails.logger.info('Simple forms api - 21-0966 submission user identity', identity:, confirmation_number:)
+      Rails.logger.info('Simple forms api - 21-0966 submission benefit types', benefit_types:, confirmation_number:)
```

Comment:
I debated with myself about whether I should combine these two logs but ultimately decided on  keeping them separate. They do both contain the `confirmation_number`, though, which would help us if we ever wanted to combine them somehow downstream.

Reply:
Is confirmation_number in any way considered PII or sensitive data?

Reply:
Nope, the `confirmation_number` is assigned to each submission and is unrelated to the user's SSN or ICN (both of which are PII). In our db we _can_ get PII from the `confirmation_number` but it itself is not sensitive.